railties/lib/rails/generators/app_base.rb

```diff
@@ -226,19 +289,19 @@ def sqlite3? # :doc:
       end
 
       def skip_active_storage? # :doc:
-        options[:skip_active_storage] || options[:skip_active_record] || options[:skip_active_job]
+        options[:skip_active_storage]
       end
 
       def skip_action_mailer? # :doc:
-        options[:skip_action_mailer] || options[:skip_active_job]
+        options[:skip_action_mailer]
       end
 
       def skip_action_mailbox? # :doc:
-        options[:skip_action_mailbox] || skip_active_storage?
+        options[:skip_action_mailbox]
       end
 
       def skip_action_text? # :doc:
-        options[:skip_action_text] || skip_active_storage?
+        options[:skip_action_text]
       end
```

Comment:
Is there a reason you are keeping these 3 methods (`skip_action_mailer?`, `skip_action_mailbox?`, and `skip_action_text?`)? This pr removes them from the `include_all_railties?` definition (above), and I don't see anywhere else in the codebase where they are actually called...

(`skip_active_storage?` could now also be replaced with a simple options check--as is done for all the other options--but that would require updating the eleven or so times where it is called.)

Reply:
The `# :doc:` comments here indicate that these methods are part of the public API, so we shouldn't remove them without deprecating them first.  (Though they don't appear in the [API documentation](https://api.rubyonrails.org/v7.0/classes/Rails/Generators/AppBase.html), but that is a separate issue. :sweat_smile:.)

Reply:
👍